pkg/parse/namespace.go

```diff
@@ -248,16 +251,46 @@ func (p *namespace) setRenderingStatusWithRetires(ctx context.Context, newStatus
 	return nil
 }
 
+// SyncStatus implements the Parser interface
+// SyncStatus gets the RootSync sync status.
+func (p *namespace) LoadStatus(ctx context.Context) (*ReconcilerStatus, error) {
```

Comment:
I dont love "Load" but I couldnt think of something better. I didn't want to just use "Status" or "GetStatus" since it makes an API call, but maybe that's more of an implementation detail.

The Parser interface method names are kind of a mess...

Reply:
Maybe `ReconcilerStatusFromCluster`?

Reply:
Renamed